app/shared/src/forms/datePicker/DatePicker.android.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import * as React from 'react';
+import { DatePickerAndroid } from 'react-native';
+
+import DatePickerButton from './DatePickerButton';
+
+type Props = {
+  date: Date,
+  format?: string,
+  onDateChange: (date: Date) => void,
+  minDate?: Date,
+  maxDate?: Date,
```

Comment:
I prefer to keep required params first so it's clear that you need to use `date` and `onDateChange`. What do you think?